uplink/metainfo/client.go

```diff
@@ -223,3 +224,21 @@ func (metainfo *Metainfo) ListSegments(ctx context.Context, bucket string, prefi
 
 	return items, response.GetMore(), nil
 }
+
+// ValueAttributeInfo requests the connector key info
+func (metainfo *Metainfo) ValueAttributeInfo(ctx context.Context, bucket string, path storj.Path, segmentIndex int64, partnerID, userID string) (pointer *pb.Pointer, err error) {
```

Comment:
Why this method returns `pointer`? Is this needed for partner/connector case?

Reply:
Wanted to be returned some value, and did a best guess of returning pointer. On a second thought changed it to more application specific and returned bool.

Reply:
Maybe just return empty response, if no error occur it will mean it's ok

Reply:
yep made suggested change